homeassistant/components/androidtv/media_player.py

```diff
@@ -587,6 +583,20 @@ def adb_command(self, cmd):
         self.schedule_update_ha_state()
         return self._adb_response
 
+    @adb_decorator()
+    def learn_sendevent(self):
+        """Translate a key press on a remote to ADB 'sendevent' commands."""
+        output = self.aftv.learn_sendevent()
+        if output:
+            self._adb_response = output
+            self.schedule_update_ha_state()
+
+            msg = f"Output from service '{SERVICE_LEARN_SENDEVENT}' from {self.entity_id}: '{output}'"
+            self.hass.components.persistent_notification.async_create(
```

Comment:
We can't use the async api in sync context.

Reply:
But it's a sync function, so it's OK, right?

https://github.com/home-assistant/core/blob/dev/homeassistant/components/persistent_notification/__init__.py#L67-L86

Reply:
It's not a sync function. It should run in the event loop so must be called from async context. We're in sync context here.

Reply:
I can address this later today. Should I simply use `hass.services.call` to create the persistent notification?

Reply:
This should work:
```suggestion
            self.hass.components.persistent_notification.create(
```